server/datastore/psql/create_maintenance_requests.go

```diff
@@ -0,0 +1,33 @@
+package psql
+
+import (
+    "time"
+    "github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+func (db *DB) CreateMaintenanceRequests(request string, userID int, time time.Time) (int, error) {
+    var dbRequest datastore.MaintenanceRequest
+    lastID := 0
```

Comment:
You don't need to figure out the ID. Postgres will automatically update the ID. So you can just do an insert with `request`, `user_submitted`, and `date_submitted`.

Reply:
fix: changed migration--id is now type `serial` for psql to automatically update the id